### PR TITLE
Update vagrant launch steps to access UI

### PR DESCRIPTION
**Pull request by kmova**
This is to help with #222 and #224 

Provide detailed instructions for setting up local vagrant clusters using vagrant. (This page can be referenced from docs)

### CODE REVIEW

File: k8s/vagrant/README.md [hunk 1]
```diff
@@ -1,27 +1,36 @@
...
 ```
 1.Vagrant (>=1.9.1)
 2.VirtualBox 5.1
-3.curl or wget or git, etc., to download the Vagrant file. 
 ```
 
 ## Download and Verify 
 
-Dowload the required Vagrantfile. You can use wget to download a specific vagrant file, like shown below - or you can just close the repository locally. 
+Download the required Vagrantfile. Use curl or wget or git, etc., to download the required Vagrantfile. This example uses wget.
```
Comment (mahebbar) [inline k8s/vagrant/README.md]: Use curl, wget, git and so on, ....

File: k8s/vagrant/README.md [hunk 2]
```diff
@@ -1,27 +1,36 @@
...
+- Download Vagrantfile
+- Bringup K8s Cluster
+- Install kubectl on the host 
+- Setup access to Kubernetes UI/Dashboard (Starting from Vagrantfile 1.7.5)
+- Setup OpenEBS
+- Launch demo pod
+
+*Note: The instructions are from an Ubuntu 16.06 host.*
 
 ## Prerequisites
```
Comment (mahebbar) [inline k8s/vagrant/README.md]: Prerequisites:

File: k8s/vagrant/README.md [hunk 3]
```diff
@@ -1,27 +1,36 @@
...
 
-This Vagrantfile can be used on laptop or Baremetal server installed with Ubuntu 16.04 and Virtualization Enabled
+This Vagrantfile can be used on any machine with Virtualization Enabled, like laptop or baremetal server. 
 
+The below instructions will walk you through the following:
+- Verify Prerequisites
+- Download Vagrantfile
+- Bringup K8s Cluster
+- Install kubectl on the host 
+- Setup access to Kubernetes UI/Dashboard (Starting from Vagrantfile 1.7.5)
```
Comment (mahebbar) [inline k8s/vagrant/README.md]: (Vagrantfile version 1.7.5 onwards)

File: k8s/vagrant/README.md [hunk 4]
```diff
@@ -1,27 +1,36 @@
...
-OpenEBS provides vagrant boxes with prepackaged kuberentes images. There are different vagrant boxes created depending on the kubernetes release. The Vagrantfiles are organized here based on the kubernetes version used by the box. A deployer can choose the Vagrantfile, based on his choice of kubernetes version. 
+OpenEBS provides vagrant boxes with prepackaged kuberentes images. There are different vagrant boxes created depending on the kubernetes release. The Vagrantfiles are organized here based on the kubernetes version used by the box. 
 
-This Vagrantfile can be used on laptop or Baremetal server installed with Ubuntu 16.04 and Virtualization Enabled
+This Vagrantfile can be used on any machine with Virtualization Enabled, like laptop or baremetal server. 
 
+The below instructions will walk you through the following:
+- Verify Prerequisites
+- Download Vagrantfile
+- Bringup K8s Cluster
```
Comment (mahebbar) [inline k8s/vagrant/README.md]: Setup K8s cluster

File: k8s/vagrant/README.md [hunk 5]
```diff
@@ -1,27 +1,36 @@
 # Installing Kubernetes Clusters on Ubuntu 16.04 using Vagrant
 
-OpenEBS provides vagrant boxes with prepackaged kuberentes images. There are different vagrant boxes created depending on the kubernetes release. The Vagrantfiles are organized here based on the kubernetes version used by the box. A deployer can choose the Vagrantfile, based on his choice of kubernetes version. 
+OpenEBS provides vagrant boxes with prepackaged kuberentes images. There are different vagrant boxes created depending on the kubernetes release. The Vagrantfiles are organized here based on the kubernetes version used by the box. 
 
-This Vagrantfile can be used on laptop or Baremetal server installed with Ubuntu 16.04 and Virtualization Enabled
+This Vagrantfile can be used on any machine with Virtualization Enabled, like laptop or baremetal server. 
 
+The below instructions will walk you through the following:
+- Verify Prerequisites
```
Comment (mahebbar) [inline k8s/vagrant/README.md]: prerequisites

File: k8s/vagrant/README.md [hunk 6]
```diff
@@ -1,27 +1,36 @@
 # Installing Kubernetes Clusters on Ubuntu 16.04 using Vagrant
 
-OpenEBS provides vagrant boxes with prepackaged kuberentes images. There are different vagrant boxes created depending on the kubernetes release. The Vagrantfiles are organized here based on the kubernetes version used by the box. A deployer can choose the Vagrantfile, based on his choice of kubernetes version. 
+OpenEBS provides vagrant boxes with prepackaged kuberentes images. There are different vagrant boxes created depending on the kubernetes release. The Vagrantfiles are organized here based on the kubernetes version used by the box. 
 
-This Vagrantfile can be used on laptop or Baremetal server installed with Ubuntu 16.04 and Virtualization Enabled
+This Vagrantfile can be used on any machine with Virtualization Enabled, like laptop or baremetal server. 
 
+The below instructions will walk you through the following:
```
Comment (mahebbar) [inline k8s/vagrant/README.md]: Procedures listed in this section helps you -

File: k8s/vagrant/README.md [hunk 7]
```diff
@@ -1,27 +1,36 @@
 # Installing Kubernetes Clusters on Ubuntu 16.04 using Vagrant
 
-OpenEBS provides vagrant boxes with prepackaged kuberentes images. There are different vagrant boxes created depending on the kubernetes release. The Vagrantfiles are organized here based on the kubernetes version used by the box. A deployer can choose the Vagrantfile, based on his choice of kubernetes version. 
+OpenEBS provides vagrant boxes with prepackaged kuberentes images. There are different vagrant boxes created depending on the kubernetes release. The Vagrantfiles are organized here based on the kubernetes version used by the box. 
 
-This Vagrantfile can be used on laptop or Baremetal server installed with Ubuntu 16.04 and Virtualization Enabled
+This Vagrantfile can be used on any machine with Virtualization Enabled, like laptop or baremetal server.
```
Comment (mahebbar) [inline k8s/vagrant/README.md]: virtualization *Enabled* (italics)

File: k8s/vagrant/README.md [hunk 8]
```diff
@@ -1,27 +1,36 @@
...
+- Install kubectl on the host 
+- Setup access to Kubernetes UI/Dashboard (Starting from Vagrantfile 1.7.5)
+- Setup OpenEBS
+- Launch demo pod
+
+*Note: The instructions are from an Ubuntu 16.06 host.*
 
 ## Prerequisites
 
 Verify that you have the following required software installed on your Ubuntu 16.04 machine:
```
Comment (mahebbar) [inline k8s/vagrant/README.md]: Verify that you have the following software installed on your Ubuntu 16.04 machine:

File: k8s/vagrant/README.md [hunk 9]
```diff
@@ -1,27 +1,36 @@
...
 
-Dowload the required Vagrantfile. You can use wget to download a specific vagrant file, like shown below - or you can just close the repository locally. 
+Download the required Vagrantfile. Use curl or wget or git, etc., to download the required Vagrantfile. This example uses wget.
 
 ```
 mkdir k8s-demo
 cd k8s-demo
-wget https://raw.githubusercontent.com/openebs/openebs/master/k8s/vagrant/1.6/Vagrantfile
+wget https://raw.githubusercontent.com/openebs/openebs/master/k8s/vagrant/1.7.5/Vagrantfile
 vagrant status
```
Comment (mahebbar) [inline k8s/vagrant/README.md]: Vagrant Status

File: k8s/vagrant/README.md [hunk 10]
```diff
@@ -49,4 +58,69 @@ Just use *vagrant up* to bring up the cluster.
 ubuntu-host:~/k8s-demo$ vagrant up
```
Comment (mahebbar) [inline k8s/vagrant/README.md]: ..... Vagrant Up

File: k8s/vagrant/README.md [hunk 11]
```diff
@@ -49,4 +58,69 @@ Just use *vagrant up* to bring up the cluster.
...
 ```
 
+### Verify
+
+You should see output similar to this:
```
Comment (mahebbar) [inline k8s/vagrant/README.md]: The output displayed will be similar to the following:

File: k8s/vagrant/README.md [hunk 12]
```diff
@@ -49,4 +58,69 @@ Just use *vagrant up* to bring up the cluster.
...
+kubeminion-01             running (virtualbox)
+kubeminion-02             running (virtualbox)
+
+This environment represents multiple VMs. The VMs are all listed
+above with their current state. For more information about a specific
+VM, run `vagrant status NAME`.
+kiran@kmaya:~/k8s-demo$ 
+```
+
+## Install kubectl on the host
```
Comment (mahebbar) [inline k8s/vagrant/README.md]: Install kubectl on the Host

File: k8s/vagrant/README.md [hunk 13]
```diff
@@ -49,4 +58,69 @@ Just use *vagrant up* to bring up the cluster.
...
+
+Please follow the instructions for [install kubectl from binary](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-binary-via-curl).
```
Comment (mahebbar) [inline k8s/vagrant/README.md]: Follow the procedures for.....

File: k8s/vagrant/README.md [hunk 14]
```diff
@@ -49,4 +58,69 @@ Just use *vagrant up* to bring up the cluster.
...
+```
+kiran@kmaya:~/k8s-demo$ kubectl version
+Client Version: version.Info{Major:"1", Minor:"7", GitVersion:"v1.7.5", GitCommit:"17d7182a7ccbb167074be7a87f0a68bd00d58d97", GitTreeState:"clean", BuildDate:"2017-08-31T09:14:02Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"linux/amd64"}
+The connection to the server localhost:8080 was refused - did you specify the right host or port?
+kiran@kmaya:~/k8s-demo$ 
+```
+
+The connection error is expected. The next step will configure the kubectl to contact the kubernetes cluster. 
+
+## Configure kube-config from the installed cluster
```
Comment (mahebbar) [inline k8s/vagrant/README.md]: Configure kube-config from the Installed Cluster

Comment (kmova) [reply k8s/vagrant/README.md]: done

File: k8s/vagrant/README.md (hunk 13 above)
Comment (kmova) [reply k8s/vagrant/README.md]: done

File: k8s/vagrant/README.md (hunk 12 above)
Comment (kmova) [reply k8s/vagrant/README.md]: done

File: k8s/vagrant/README.md (hunk 11 above)
Comment (kmova) [reply k8s/vagrant/README.md]: done

File: k8s/vagrant/README.md (hunk 10 above)
Comment (kmova) [reply k8s/vagrant/README.md]: the command is *vagrant up* -- with lowercase "v". Leaving it as is..

File: k8s/vagrant/README.md (hunk 9 above)
Comment (kmova) [reply k8s/vagrant/README.md]: the command is *vagrant status* -- with lowercase "v". Leaving it as is.

File: k8s/vagrant/README.md (hunk 8 above)
Comment (kmova) [reply k8s/vagrant/README.md]: done

File: k8s/vagrant/README.md (hunk 7 above)
Comment (kmova) [reply k8s/vagrant/README.md]: done

File: k8s/vagrant/README.md (hunk 6 above)
Comment (kmova) [reply k8s/vagrant/README.md]: done

File: k8s/vagrant/README.md (hunk 5 above)
Comment (kmova) [reply k8s/vagrant/README.md]: done

File: k8s/vagrant/README.md (hunk 4 above)
Comment (kmova) [reply k8s/vagrant/README.md]: done

File: k8s/vagrant/README.md (hunk 3 above)
Comment (kmova) [reply k8s/vagrant/README.md]: done

File: k8s/vagrant/README.md (hunk 2 above)
Comment (kmova) [reply k8s/vagrant/README.md]: done

File: k8s/vagrant/README.md (hunk 1 above)
Comment (kmova) [reply k8s/vagrant/README.md]: done

**Comment by kmova** [comment]
Made this global change -- instances of kubernetes capitalized (Kubernetes)

Leaving box(es) unchanged. Vagrant terminology uses boxes.

**Pull request merged**